src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -84,7 +86,13 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
```

Comment:
Thanks for catching this, but please make this change in a separate commit `minor: formatting`

Reply:
Okay, I am doing it.

Reply:
Changes Done. Created new Commit.

Reply:
Please make message of separate commit like example I gave above. Do not reference this issue, your change has nothing to do with it. This is why we are making a separate commit.

Reply:
Also, your other commit should have only the change on this line in it, nothing else.

Reply:
Changes Done

Reply:
Not done, put only the change of indentation of closing `}` in separate commit.

Reply:
@nick-mancuso Just to clarify: I have to create a new commit in which `line 95:3` should just be formatted with `95:5`

Reply:
The closing brace of the method in this example was not indented correctly before your changes. While adding your changes to this example, you fixed the indentation of this closing brace. Fixing this indentation is not related to this issue. Any time that we make changes that are not related to the issue we work on, we do this in a separate commit. We do this to make review easier, and for future maintainers to have an easier time understanding changes that were made regarding a specific issue.

Reply:
Please checkout now. I have done changes.

Reply:
Not done. 

In first commit, make no changes to line 95. Only add to the code example (all changes “+” only in this code example).

In second commit, fix indentation of closing brace on line 95.

Reply:
done

Reply:
Resolved.